### ISSUE TITLE
Should give a warning if deployment fails

### ISSUE THREAD

**Issue by georgejecook** [TwitchBronBron/vscode-brightscript-language#69]
Many of us change roku accounts frequently. When this happens, we might forget to enable debug mode, or the box might crash and not accept more deployments.

In any case, it'd be great to have an error/warning if the deployment is unsuccessful.

**Comment by TwitchBronBron** [comment]
What type of presentation do you think should should show for this? My top choice right now is one of those notifications at the bottom right of the screen. Like this one (but with error or warning status)

![image](https://user-images.githubusercontent.com/2544493/51178845-d2f35e00-1891-11e9-974b-729ded05775f.png)

**Comment by georgejecook** [comment]
That’d make sense - it’d be consistent with other extensions I use.


On January 15, 2019 at 6:50:30 AM, Bronley Plumb (notifications@github.com)
wrote:

What type of presentation do you think should should show for this? My top
choice right now is one of those notifications at the bottom right of the
screen. Like this one (but with error or warning status)

[image: image]
<https://user-images.githubusercontent.com/2544493/51178845-d2f35e00-1891-11e9-974b-729ded05775f.png>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/TwitchBronBron/vscode-brightscript-language/issues/69#issuecomment-454366079>,
or mute the thread

**Comment by TwitchBronBron** [comment]
I found a few places where the debug failure errors were not being bubbled up to the top, so I fixed those in [this branch](https://github.com/TwitchBronBron/vscode-brightscript-language/tree/feature/better-failure-presentation). I tested out a few other debuggers, and they all use the standard vscode debugger failure path, which is to call ` this.sendErrorResponse`. That shows an alert box in the center of the screen. I'd be open to discussing alternatives, but in general I would like to stick to using the vscode standards wherever possible. Thoughts?

![image](https://user-images.githubusercontent.com/2544493/51316975-4c22ba80-1a24-11e9-9a7a-3ddc0c4d65a6.png)
